src/site/xdoc/download_collections.xml

```diff
@@ -208,27 +208,27 @@ limitations under the License.
-              <td><a href="https://www.apache.org/dist/commons/collections/binaries/commons-collections-3.2.2-bin.tar.gz.sha256">sha256</a></td>
-              <td><a href="https://www.apache.org/dist/commons/collections/binaries/commons-collections-3.2.2-bin.tar.gz.asc">pgp</a></td>
+              <td><a href="https://downloads.apache.org/commons/collections/binaries/commons-collections-3.2.2-bin.tar.gz.sha256">sha256</a></td>
+              <td><a href="https://downloads.apache.org/commons/collections/binaries/commons-collections-3.2.2-bin.tar.gz.asc">pgp</a></td>
           </tr>
           <tr>
               <td><a href="[preferred]/commons/collections/binaries/commons-collections-3.2.2-bin.zip">commons-collections-3.2.2-bin.zip</a></td>
-              <td><a href="https://www.apache.org/dist/commons/collections/binaries/commons-collections-3.2.2-bin.zip.sha256">sha256</a></td>
-              <td><a href="https://www.apache.org/dist/commons/collections/binaries/commons-collections-3.2.2-bin.zip.asc">pgp</a></td>
+              <td><a href="https://downloads.apache.org/commons/collections/binaries/commons-collections-3.2.2-bin.zip.sha256">sha256</a></td>
+              <td><a href="https://downloads.apache.org/commons/collections/binaries/commons-collections-3.2.2-bin.zip.asc">pgp</a></td>
           </tr>
         </table>
       </subsection>
       <subsection name="Source">
         <table>
           <tr>
               <td><a href="[preferred]/commons/collections/source/commons-collections-3.2.2-src.tar.gz">commons-collections-3.2.2-src.tar.gz</a></td>
-              <td><a href="https://www.apache.org/dist/commons/collections/source/commons-collections-3.2.2-src.tar.gz.sha256">sha256</a></td>
-              <td><a href="https://www.apache.org/dist/commons/collections/source/commons-collections-3.2.2-src.tar.gz.asc">pgp</a></td>
+              <td><a href="https://downloads.apache.org/commons/collections/source/commons-collections-3.2.2-src.tar.gz.sha256">sha256</a></td>
+              <td><a href="https://downloads.apache.org/commons/collections/source/commons-collections-3.2.2-src.tar.gz.asc">pgp</a></td>
           </tr>
           <tr>
               <td><a href="[preferred]/commons/collections/source/commons-collections-3.2.2-src.zip">commons-collections-3.2.2-src.zip</a></td>
-              <td><a href="https://www.apache.org/dist/commons/collections/source/commons-collections-3.2.2-src.zip.sha256">sha256</a></td>
-              <td><a href="https://www.apache.org/dist/commons/collections/source/commons-collections-3.2.2-src.zip.asc">pgp</a></td>
+              <td><a href="https://downloads.apache.org/commons/collections/source/commons-collections-3.2.2-src.zip.sha256">sha256</a></td>
+              <td><a href="https://downloads.apache.org/commons/collections/source/commons-collections-3.2.2-src.zip.asc">pgp</a></td>
```

Comment:
The old links appear to still work, but maybe it's fine to update this file manually.

It is automatically generated by the commons-build-plugin: https://github.com/apache/commons-build-plugin/tree/master/src/main/resources/commons-xdoc-templates

Reply:
File has been rollbacked.